untangle-linux-config/debian/postinst

```diff
@@ -2,6 +2,12 @@
 
 set -x
 
+bin_sym_link=/bin
+# NGFW-13296 create /bin symlink if missing/broken
+if [ ! -L ${bin_sym_link} ] || [ ! -e ${bin_sym_link} ] ; then
+  ln -s /usr/bin ${bin_sym_link}
```

Comment:
Double check this.  On my system I see the symlink set as :
ls -lad /bin
lrwxrwxrwx 1 root root 7 Jul 26  2022 /bin -> usr/bin
So it may be:
cd /
ln -s usr/bin /bin
cd -

Reply:
Hmm good catch.